src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,52 @@
+.nav {
+  display: block;
+  margin-right: 50px;
+  @extend %text-font-12;
```

Comment:
I dont know if it is necessery. It should inherit this placeholder from html tag